emby_client.py

```diff
@@ -127,9 +127,9 @@ def instant_mix(self, item_id):
         return self._get(instant_item_mix)
 
     def get_song_file(self, song_id):
-        url = '{0}{1}/{2}/{3}?userId={4}&MaxStreamingBitrate=140000000&AudioCodec=mp3'\
+        url = '{0}{1}/{2}/{3}?userId={4}&API_KEY={5}&MaxStreamingBitrate=140000000&AudioCodec=mp3'\
```

Comment:
@fnorbenden, 
I had to add back in this token so when mycroft hits the generated song url it is authenticated to stream the song.